.circleci/config.yml

```diff
@@ -141,16 +141,11 @@ jobs:
       - run: md test-results
 
       - run:
-          name: "List rules"
-          command: bin\run.cmd scanner:rule:list
+          name: "Unpackaged sanity test"
+          command: smoke-tests\powershell.cmd bin\run.cmd
```

Comment:
Is the "bin\run.cmd" only temporary until we create the tarballl? 

What is the advantage of running both "bin\run.cmd" and "sfdx" commands?

Reply:
Once we create the tarball, we can replace `bin\run.cmd` with the tarball invocation. But right now, `bin\run.cmd` is how we invoke the local build, and `sfdx` is how we invoke the installed version.